controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/CommitRequestHandler.java

```diff
@@ -148,6 +149,7 @@ public CompletableFuture<Void> execute(CommitEvent event) {
                         if (r >= 0) {
                             log.info(requestId, "Successfully committed transactions on epoch {} on stream {}/{}", r, 
                                     scope, stream);
+                            TransactionMetrics.getInstance().controllerEventProcessorCommitTransactionLatency(timer.getElapsed());
```

Comment:
Is the indentation correct?

Reply:
what is there any indentation issue here ? looks fine to me